projects/i18n/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@vcd/ui-i18n",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^8.2.9",
+    "@angular/core": "^8.2.9"
+  },
+  "bin": {
+    "create-translations": "../scripts/index.js"
```

Comment:
what is the intended usage of this?

Reply:
It was originally going to be a provided script to help plugins combine their translations with our defaults. Now, it is just going to function as an example that I'll talk about in response to Juan's comment. Removed this bin line in the most recent commit.